LibraBFT/Impl/OBM/ECP-LBFT-OBM-Diff/ECP-LBFT-OBM-Diff-1.agda

```diff
@@ -40,6 +42,43 @@ e_RoundState_processLocalTimeoutM e r =
 
 ------------------------------------------------------------------------------
 
+postulate
+ e_EpochManager_doECP_waitForRlec
+  : EpochManager → EpochChangeProof
+  → Either ErrLog Bool
+
+------------------------------------------------------------------------------
+
+e_EpochManager_startNewEpoch
+  : EpochManager → EpochChangeProof
+  → Either ErrLog EpochManager
+e_EpochManager_startNewEpoch self ecp =
+  if not ECP-LBFT-OBM-Diff-0.enabled
```

Comment:
Perhaps a bit silly for this small example, but I pushed changes to use `ifD` here, as it would be good to get into the habit of using the variants of conditionals that are compatible with the machinery supporting Dijkstra's weakest precondition calculus, following [these recommendations](https://github.com/oracle/bft-consensus-agda/blob/chris2-mark-qcsb4/docs/PeerHandlerContracts.org#eitherd-and-monadic-bind).  This led me to explore some ways to streamline the boilerplate for this approach, which I will share in a separate branch as it requires further discussion and evaluation.

Reply:
great - thanks